qutebrowser/browser/commands.py

```diff
@@ -682,7 +682,10 @@ def _yank_url(self, what):
         else:
             flags |= QUrl.FullyEncoded
         url = QUrl(self._current_url())
-        url_query = QUrlQuery(url)
+        url_query = QUrlQuery()
+        if ';' in url.query():
+            url_query.setQueryDelimiters('=', ';')
```

Comment:
Good point. Actually, neither (and a few more) should be fiddled with at all, but checking for `'&' not in`, perhaps in addition, should be more robust in practice.